stdlib/public/core/StringGuts.swift

```diff
@@ -50,8 +50,9 @@ extension _StringGuts {
     self.init(_StringObject(smol))
   }
 
+  @_alwaysEmitIntoClient
   @inlinable @inline(__always)
-  internal init(_ bufPtr: UnsafeBufferPointer<UInt8>, isASCII: Bool) {
+  internal init(immortal bufPtr: UnsafeBufferPointer<UInt8>, isASCII: Bool) {
     self.init(_StringObject(immortal: bufPtr, isASCII: isASCII))
   }
```

Comment:
Is the `@_alwaysEmitIntoClient` needed for availability?

Reply:
For back-deployment, yes.